File: back/src/main/java/com/tdevilleduc/urthehero/back/dao/StoryDao.java
package com.tdevilleduc.urthehero.back.dao;

import com.tdevilleduc.urthehero.model.Story;
import org.springframework.data.jpa.repository.JpaRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface StoryDao extends JpaRepository<Story, Integer> {

    Story findById(int id);
}

File: back/src/main/java/com/tdevilleduc/urthehero/back/dao/StoryDao.kt
package com.tdevilleduc.urthehero.back.dao

import com.tdevilleduc.urthehero.back.model.Story
import org.springframework.data.jpa.repository.JpaRepository
import org.springframework.stereotype.Repository

@Repository
interface StoryDao : JpaRepository<Story, Int>